src/java.base/linux/native/libnio/fs/LinuxNativeDispatcher.c

```diff
@@ -36,6 +38,10 @@
 
 #include "sun_nio_fs_LinuxNativeDispatcher.h"
 
+#ifndef FICLONE
+#define FICLONE      1074041865
```

Comment:
Are we sure this value is the same on all architectures?

Reply:
Testing succeeded on all platforms I've tried. It's been around since [Linux 4.5](https://man7.org/linux/man-pages/man2/ioctl_ficlone.2.html#VERSIONS) which was released [13 March 2016](https://en.wikipedia.org/wiki/Linux_kernel_version_history#Releases_4.x.y).